packages/appeals-service-api/src/services/appeal.service.js

```diff
@@ -122,10 +122,26 @@ async function updateAppeal(id, appealUpdate) {
 	Object.assign(appeal, appealUpdate);
 	isValidAppeal(appeal);
 
-	/* eslint no-param-reassign: ["error", { "props": false }] */
 	appeal.updatedAt = new Date(new Date().toISOString());
 	const updatedAppealEntity = await appealsCosmosRepository.update(appeal);
 	const updatedAppeal = updatedAppealEntity.value.appeal;
+
+	if (appealUpdate.state || appealUpdate.decisionDate) {
+		/** 
+		 	@type Partial<import('../repositories/sql/appeals-repository').AppealCreateInput>
+		*/
+		let sqlUpdate = {
+			legacyAppealSubmissionId: id
+		};
+		if (appealUpdate.decisionDate) {
+			sqlUpdate.legacyAppealSubmissionDecisionDate = appealUpdate.decisionDate;
+		}
+		if (appealUpdate.state) {
+			sqlUpdate.legacyAppealSubmissionState = appealUpdate.state;
+		}
+		appealsSQLRepository.updateAppealByLegacyAppealSubmissionId(sqlUpdate);
+	}
```

Comment:
I don't know the codebase well enough to know if we do it: but if these fields were updated to `null` or something, this wouldn't work.

Perhaps `hasOwnProperty` check or something instead of checking for truthy values?

Reply:
haha I'd written this to purposefully not make them null 😅 I'll rejig it